build.gradle

```diff
@@ -0,0 +1,681 @@
+apply plugin: 'war'
+apply plugin: 'idea'
+apply plugin: 'java'
```

Comment:
Thanks for the tip.  I will update the build.gradle file.  I have also made a note to research jacoco.  Thank you.